stewards.json

```diff
@@ -1 +1,7 @@
-[]
+[
+    {
+        "ca": "standard_crypto/server_ca.crt",
+        "ca_hosted": "https://steward.sommelier.standardcryptovc.com/server_ca.crt",
+        "endpoint": "https://steward.sommelier.standardcryptovc.com:5784"
```

Comment:
Did you mean 5734? There was a typo in the readme that's since been fixed.

Reply:
Thank you!